tests/Database/SelectFields/ArrayTests/PostType.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rebing\GraphQL\Tests\Database\SelectFields\ArrayTests;
+
+use GraphQL\Type\Definition\Type;
+use Rebing\GraphQL\Support\Facades\GraphQL;
+use Rebing\GraphQL\Tests\Support\Models\Post;
+use Rebing\GraphQL\Support\Type as GraphQLType;
+
+class PostType extends GraphQLType
+{
+    protected $attributes = [
+        'name' => 'Post',
+        'model' => Post::class,
+    ];
+
+    public function fields(): array
+    {
+        return [
+            'id' => [
+                'type' => Type::nonNull(Type::ID()),
+            ],
+            'properties' => [
+                'type' => Type::listOf(GraphQL::type('Property')),
+                'selectable' => false,
```

Comment:
Replace this with `'is_relation' => false,` => this is also your fix for https://github.com/rebing/graphql-laravel/issues/416 I believe!